### ISSUE TITLE
add support for .jshintignore file

### ISSUE THREAD

**Issue by SET001** [victorporof/Sublime-JSHint#113]
Add support for `.jshintignore` file. Without this there are no way to set ignored files.
My current solution is to add `// jshint ignore: start` on the top of each file that should be ignored. But this is not as good as settings once something like `spec/*.js` in `.jshintignore`.

**Comment by victorporof** [comment]
Sorry for taking so long to get to this. Is it possible to rebase your changes to the current tip? Otherwise I'll look into it. Thank you!

**Comment by arronmabrey** [comment]
@SET001 Yes this would be awesome. I actually have an issue where I'm using script tags for templating in an index.html file. e.g.
```
<script type="text/template">
...HTML MARKUP...
</script>
```
Obviously JSHint goes crazy trying to "hint" html markup in that script tag. It would be great to ignore these html files.

**Comment by jhinzmann** [comment]
A comment on my pull-request #119 by @austil got my attention on this (maybe a bit late, but better than never).
I just rebased my changes onto the current master without any problems.
